grpc/authn-private.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package grpc;
```

Comment:
More unsolicited feebback, sorry! A package name like `keratin/authn` will lead to better looking code in users calling your server. Also, it will prevent collisions in the types, because for anyhow using your gRPC client, there's a global namespace of Protobuf types that are registered by `$package_name` and `$message_name`. So if your package name is just `grpc`, that's more likely to collide with other protobuf types that your users have in their apps.